design_drafts/components_architecture_and_urdf_examples.md

```diff
@@ -415,285 +414,281 @@ Note:
       <param name="read_for_sec">0.001</param>
     </hardware>
     <joint name="front_left_leg_joint1">
-      <commandInterfaceType name="position">
+      <command_interface name="position">
         <param name="min">-1</param>
         <param name="max">1</param>
-      </commandInterfaceType>
-      <commandInterfaceType name="velocity">
+      </command_interface>
+      <command_interface name="velocity">
         <param name="min">-1</param>
         <param name="max">1</param>
-      </commandInterfaceType>
-      <commandInterfaceType name="effort">
+      </command_interface>
+      <command_interface name="effort">
         <param name="min">-0.5</param>
         <param name="max">0.5</param>
-      </commandInterfaceType>
-      <commandInterfaceType>Kp</commandInterfaceType>
-      <commandInterfaceType>Kd</commandInterfaceType>
-      <commandInterfaceType>effort_limit</commandInterfaceType>
-      <stateInterfaceType>position</stateInterfaceType>
-      <stateInterfaceType>velocity</stateInterfaceType>
-      <stateInterfaceType>effort</stateInterfaceType>
-      <stateInterfaceType>coil_resistance</stateInterfaceType>
-      <stateInterfaceType>timestamp</stateInterfaceType>
+      </command_interface>
+      <command_interface name="Kp"/>
```

Comment:
Do we need here "min/max" parameters?
I think we defined them as mandatory for the `command_interfaces`...

Reply:
Ok, they are not mandatory...